### DISCUSSION TITLE
Relations among issues

### DISCUSSION THREAD

**Discussion by Julian-Dumitrascu** [community/community#73536] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

1. One can use a project to create a type of items / issues.
With other software (e.g. Coda) one can manage relations among such items.
We ask for the ability to pick in a field any number of issues from any repositories. One must be able to use any number of fields of this type in any project.
I am willing to discuss such a feature sufficiently, so that most participants get results they want.

2. I’d like to talk with a Microsoft relationship manager about the relationship between the business group Sol Global Management and Microsoft.
We can discuss e.g. benefits you seek and how to shape some Microsoft services, e.g. GitHub, so that you enjoy these benefits to the highest degree. Microsoft is one of the most important software developers; at the same time, it has room for development: for instance, we can check what reasons people have to pay for software similar to software made by Microsoft. One can say that it is good that GitHub earns USD 10^9 per year, reached 0.1 billion users in 15 years, and grew the most after being acquired by Microsoft (2018); at the same time, GitHub deserves more users. 
Who cares the most about the next 5 years?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by sschuberth** [comment]
See https://github.com/orgs/community/discussions/4928.
